### ISSUE TITLE
Don't use camel_case_linter on package names

### ISSUE THREAD

**Issue by bmwilly** [jimhester/lintr#201]
Reproducible example: 

```bash
$ echo "library(randomForest)" > bad.R
$ Rscript -e "lintr::lint('bad.R')"
bad.R:1:9: style: Variable and function names should be all lowercase.
library(randomForest)
        ^~~~~~~~~~~~
```

Strangely, this doesn't happen if only the first letter is capitalized: 
```bash
$ echo "library(Rcpp)" > good.R
$ Rscript -e "lintr::lint('good.R')"
$ 
```

**Comment by jfiksel** [comment]
I'm having the same issue, would love to see this fixed

**Comment by fangly** [comment]
You could use object_name_linter in the meantime (https://github.com/jimhester/lintr/issues/194).

**Comment by fangly** [comment]
Fixed in #214
